components/datadog/dockeragentparams/params.go

```diff
@@ -130,10 +130,10 @@ func withIntakeHostname(hostname pulumi.StringInput, shouldSkipSSLCertificateVal
 	}).(pulumi.BoolOutput)
 	return func(p *Params) error {
 		envVars := pulumi.Map{
-			"DD_DD_URL":                                 pulumi.Sprintf("http://%s:80", hostname),
+			"DD_DD_URL":                                 pulumi.Sprintf("https://%s", hostname),
 			"DD_LOGS_CONFIG_DD_URL":                     pulumi.Sprintf("%s:80", hostname),
```

Comment:
Should the log intake be also migrated to http**s** ?

```suggestion
			"DD_LOGS_CONFIG_DD_URL":                     pulumi.Sprintf("%s:443", hostname),
```

Reply:
Is it mandatory to specify the port ?